enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,180 @@
+---
+title: targeted-update-edge-blocking
+authors:
+  - "@wking"
+reviewers:
+  - "@dofinn"
+  - "@LalatenduMohanty"
+  - "@sdodson"
+  - "@steveeJ"
+  - "@vrutkovs"
+approvers:
+  - TBD
+creation-date: 2020-07-07
+last-updated: 2020-08-07
+status: implementable
+---
+
+# Targeted Update Edge Blocking
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes a mechanism for blocking edges for the subset of clusters considered vulnerable to known issues with the stability of a particular update or target release.
+
+## Motivation
+
+When managing the [Cincinnati][cincinnati-spec] update graph [for OpenShift][cincinnati-for-openshift-design], we sometimes discover stability issues with particular release images or updates between them.
```

Comment:
We can also block on things that are more on the feature side, e.g. "deciding not to send folks over this minor bump yet, because updates will be smoother if this heavy RHCOS pull happens in a patch bump".  Changing to the very generic `issues` with 2037152 -> b78cced.